packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -143,88 +156,94 @@ String addServiceExtensions() {
                     });
                   }
                   additionalJson['constraints'] = constraintsProperty;
+                } catch (e) {
+                  // Not laid out yet.
                 }
                 if (renderObject is RenderBox) {
                   additionalJson['size'] = <String, Object>{
                     'width': renderObject.size.width.toString(),
                     'height': renderObject.size.height.toString(),
                   };
 
-                  final ParentData parentData = renderObject.parentData;
+                  final ParentData parentData =
+                      renderObject.parentData as ParentData;
                   if (parentData is FlexParentData) {
-                    additionalJson['flexFactor'] = parentData.flex;
+                    additionalJson['flexFactor'] = parentData.flex as int;
                     additionalJson['flexFit'] =
                         describeEnum(parentData.fit ?? FlexFit.tight);
                   }
                 }
               }
-              return additionalJson;
-            }),
-      );
-    }
-    return Future.value(<String, Object>{
+            }
+            return additionalJson;
+          }),
+    ) as Map<String, dynamic>;
+    return Future.value(<String, dynamic>{
```

Comment:
again, why the change from Object to dynamic?

Reply:
same answer as above.